api_v2/management/commands/import.py

```diff
@@ -17,6 +17,9 @@ def add_arguments(self, parser):
                             "--dir",
                             type=str,
                             help="Directory to write files to.")
+        parser.add_argument("--debug",
```

Comment:
I've added this to improve error logging. Now it should be possible to see which file and line caused the import to fail